### ISSUE TITLE
✨  Routing for result-app

### ISSUE THREAD

**Issue by keithrfung** [microsoft/electionguard-ui#71]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Suggestion

- Add `react-router` to `result-app`
- Add a `routes` folder with a file called `MainRoutes.tsx`
- Create the following routes and pages with an H1 to denote which page you're on.
```
  <Switch>
      <Route path="/" exact>
          <Redirect to="/menu" />
      </Route>
      <Route path="/menu" component={SelectElectionPage} />
      <Route path="/:election" component={ElectionPage} />
      <Route path="/:election/results" component={ResultsPage} />
      <Route path="/:election/records" component={RecordsPage} />
      <Route path="/:election/:ballot" component={BallotConfirmationPage} />
  </Switch>
```

**Comment by PratikDey** [comment]
Hi @keithrfung , I would like to work on this issue, could you tell me more about the project?

**Comment by gagandeepp** [comment]
@keithrfung please review the PR

**Comment by alceil** [comment]
Can i work on this issue?

**Comment by keithrfung** [comment]
Sorry everyone for replying late. So many repo's and only one Keith. @gagandeepp has already done a PR so I assigned them the issue. @alceil I assigned you a different issue. 

@PratikDey Here is a run down. I will try to add a card that adds documentation to explain this. 

This project is a UI representation of the electionguard system. It is a monorepo composed of the different parts. There are at this juncture two reusable parts an api-client and a library of election ui components. There will be two applications. One to administer an election and another to display the election results and records.
